integration/conftest.py

```diff
@@ -152,7 +157,7 @@ def cleanup():
         )
         print("Waiting for flogtool to complete")
         try:
-            pytest_twisted.blockon(flog_protocol.done)
+            block_with_timeout(flog_protocol.done, reactor)
```

Comment:
It's a leftover from a debugging attempt, yes.

However—in general blocking forever makes debugging failed tests quite difficult. So having timeouts everywhere is good policy for the next time something fails, otherwise the next person will need to add them.

Reply:
Decided to leave them in.